Lexicon.md

```diff
@@ -4,19 +4,20 @@
 * __Information__: Anything that can be encoded into a __Bit String__.
-* __Environment__: __Information__ that is defined relative to a particular __Agent__. It is seperated from an __Agent__ by an __Agent Boundary__, and encompasses all content needed to process any relevant __Actions__ and __Observations__.
+* __Environment__: __Information__ that is defined relative to a particular __Actor__. It is seperated from an __Actor__ by an __Actor Boundary__, and encompasses all content needed to process any relevant __Events__.
 * __World__: Synonymous with __Environment__.
-* __Agent__: A __System__ that interacts with an __Environment__. It can consume zero or more __Observations__, and perform zero or more __Actions__. The decision of which __Actions__ to perform is made based on the __Agent's__ beliefs.
-* __Agent Boundary__: The surface that separates an __Agent__ from its __Environment__.
-* __Observation__: __Information__ that crosses the __Agent Boundary__, entering the __Agent__.
-* __Observation Space__: The set of all __Observations__ that a particular __Agent__ may observe. All __Observations__ perceived by an __Agent__ are sampled from its __Observation Space__.
+* __Actor__: __Information__ that interacts with an __Environment__ through __Events__.
+* __Actor Boundary__: The surface that separates an __Actor__ from its __Environment__.
+* __Event__: __Information__ that crosses the __Actor Boundary__.
+* __Observation__: An __Event__ in which __Information__ enters an Agent.
+* __Observation Space__: The set of all __Observations__ that a particular Agent may observe. All __Observations__ perceived by an Agent are sampled from its __Observation Space__.
 * __Object__: A set of correlated __Observations__.
 * __System__: An __Object__ composed of correlated member __Objects__.
 * __Model__: A __System__ that represents another __System__. It can be assigned a truth value (measurement) which expresses the accuracy of the representation. Examples of __Models__ include maps, blueprints, simulations, mathematical models, beliefs and claims.
 * __World State__: Given that the __Information__ in an __Environment__ may change with respect to some parameter (for example, time), __World State__ defines a static snapshot of the __Information__ within an __Environment__. In other words, a __World State__ is an __Environment__ where all __Information__ is held constant.
-* __World Model__: An __Agent__'s internal __Model__ of its __World__.
-* __Action__: __Information__ that crosses the __Agent Boundary__, exiting the __Agent__.
-* __Action Space__: The set of all __Actions__ that a particular __Agent__ may perform. All __Actions__ performed by an __Agent__ are sampled from its __Action Space__.
-* __Value__: An __Agent__'s expression of preference between two __World States__. __Value__ may favour either __State__, or may reflect indifference. __Value__ is not intrinsic, and cannot be specified in the absence of an __Agent__.
```

Comment:
Same as https://github.com/Vulcan-Academy/Vulcanomicon/pull/60/files#r225726111 . Can __Actors__ __Value__ or may only Agents?

Reply:
I would say that Agents are the type of Actor that are usefully modeled as having preferences and other kinds of beliefs (Dennett's intentional stance)

Reply:
Ok, we can discuss further in #28

Reply:
As it stands any object can be an Actor in a particular interaction, even the simplest objects. Do all objects have values?

Reply:
To clarify, with these changes we can consider an electron to be an Actor in a given interaction, but I don't think an electron has preferences.

Reply:
OK